lib/compat/wordpress-7.0/class-gutenberg-rest-templates-controller-7-0.php

```diff
@@ -0,0 +1,408 @@
+<?php
+
+/**
+ * Core class used to access templates via the REST API for WordPress 7.0.
+ *
+ * Note: no changes have been made to this class.
+ * This class extension exists only to override the core route,
+ * and to ensure the gutenberg_resolve_pattern_blocks function is used in the prepare_item_for_response method.
+ * See: https://github.com/WordPress/gutenberg/pull/72988
```

Comment:
Tiny nit: should this be a reference to this PR rather than the earlier one?

```suggestion
 * See: https://github.com/WordPress/gutenberg/pull/73477
```

Reply:
yes! i always appreciate a copy edit 🙇🏻